docs/source/techniques/ISISPowder-Tutorials.rst

```diff
@@ -210,6 +210,10 @@ found for each individual instrument in the reference document:
 
 How objects hold state in ISIS Powder
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. warning:: This is NOT relevant for PEARL. PEARL scientists should
```

Comment:
Is there now a concern that we have classes deriving from the same base interface but behaving differently? I actually think the PEARL behaviour sounds more sensible - is there any hope to harmonise this?

Reply:
This makes me a bit uncomfortable as well. I think that scripts from other instruments rely on the old behaviour (we made some changes to that effect in #21420), so I'm not sure how much I want to disrupt other instruments' workflows? I'm sort of in two minds...